### ISSUE TITLE
Radio buttons make noise of being selected, even if already selected

### ISSUE THREAD

**Issue by KatieWoe** [phetsims/gravity-force-lab#236]
**Test device**
Dell
**Operating System**
Win 10
**Browser**
Edge
**Problem description**
For https://github.com/phetsims/QA/issues/471. Not limited to Edge.
The force values radio buttons have a sound effect that plays when you select them. However, If you click on the radio button that is already selected it will make the noise as if being selected again. This needs to be done with mouse or touch, as I cannot seem to reproduce with keyboard nav. If this is intended behavior, feel free to close.
**Steps to reproduce**
1. Use mouse or touch to select the currently active radio button.

Troubleshooting information:

<details>

!!!!! DO NOT EDIT !!!!!
Name: ‪Gravity Force Lab‬
URL: https://phet-dev.colorado.edu/html/gravity-force-lab/2.2.0-rc.1/phet/gravity-force-lab_all_phet.html
Version: 2.2.0-rc.1 2020-01-28 03:08:20 UTC
Features missing: touch
Flags: pixelRatioScaling
User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/79.0.3945.130 Safari/537.36 Edg/79.0.309.71
Language: en-US
Window: 1536x718
Pixel Ratio: 2.5/1
WebGL: WebGL 1.0 (OpenGL ES 2.0 Chromium)
GLSL: WebGL GLSL ES 1.0 (OpenGL ES GLSL ES 1.0 Chromium)
Vendor: WebKit (WebKit WebGL)
Vertex: attribs: 16 varying: 30 uniform: 4096
Texture: size: 16384 imageUnits: 16 (vertex: 16, combined: 32)
Max viewport: 32767x32767
OES_texture_float: true
Dependencies JSON: {}

</details>

**Comment by zepumph** [comment]
I think we would need to ask sound designers if this is supposed to happen @emily-phet @Ashton-Morris and @jbphet are probably the ones with the answer. Is this a bug?

**Comment by emily-phet** [comment]
This is the desired behavior. Closing.
